python/foglamp/common/service_record.py

```diff
@@ -39,10 +38,10 @@ def __init__(self, s_id, s_name, s_type, s_protocol, s_address, s_port, m_port):
         self._protocol = s_protocol
         self._address = s_address
         self._port = None
-        if s_port != None:
+        if s_port is not None:
             self._port = int(s_port)
         self._management_port = int(m_port)
-        self._status = 0
+        self._status = 1
```

Comment:
I believe, default status should be 1 (when service_record is created) if no ping let monitor loop set it to 0 and unregister